tmk_core/protocol/chibios/usb_main.c

```diff
@@ -63,46 +46,28 @@ extern keymap_config_t keymap_config;
 #    define usb_lld_disconnect_bus(usbp)
 #endif
 
-uint8_t                keyboard_idle __attribute__((aligned(2)))     = 0;
```

Comment:
Just a heads up, quantum/console.c  refers to `keyboard_idle` if PROTOCOL_VUSB isn't defined. Might be worth changing it

Reply:
The idle handling got a big overhaul in this PR as well. For a short term fix I've deactivated the printing of the idle rate for now.